docs/cn/Security.md

```diff
@@ -106,6 +107,52 @@ Alluxio文件系统为目录和文件实现了一个访问权限模型，该模
 所属用户只能由超级用户修改。
 所属组和访问权限只能由超级用户和文件所有者修改。
 
+## 审查
+Alluxio支持审查日志用于系统管理员追踪用户对文件元数据的访问操作。
+
+审查日志文件(`master_audit.log`) 包括多个审查记录条目，每个条目对应一次文件元数据获取记录。
+Alluxio审查日志格式如下表所示：
+
+<table class="table table-striped">
+<tr><th>key</th><th>value</th></tr>
+<tr>
+  <td>succeeded</td>
```

Comment:
Can you also translate the content in this table?

Reply:
The table has been translated. Thanks for the comment.